charts/dns_canary/templates/cronjob.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+spec:
+  schedule: {{ .Values.schedule | quote }}
+  concurrencyPolicy: Allow
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  suspend: false
+  jobTemplate:
+    spec:
+      backoffLimit: 6
+      completions: 1
+      parallelism: 1
+      template:
+        metadata:
+          labels:
```

Comment:
This needs to use the `selector-labels` template from the helpers file like this:

```yaml
labels:
  {{- include "dns-canary.selectorLabels" . | nindent 8 }}
```